plugins/search-backend-node/api-report.md

```diff
@@ -113,6 +113,12 @@ export class LunrSearchEngineIndexer extends BatchSearchEngineIndexer {
   initialize(): Promise<void>;
 }
 
+// @public
+export class MissingIndexError extends Error {
```

Comment:
Could this extend our own `CustomErrorBase` to make it a oneliner? https://github.com/backstage/backstage/blob/df63debc7080c745763ec327a8e1b4b5fdb0147d/packages/errors/src/errors/common.ts#L50

Could even perhaps leverage `ConflictError` or `NotFoundError` in there directly, if that makes sense.

Reply:
yeah I did extend `CustomErrorBase` from the start, but couple of things that I didn't wanted to have in there e.g. combination of message and cause. So I felt like it didn't made sense to extend that in the end..